docs/troubleshooting.asciidoc

```diff
@@ -5,12 +5,12 @@ link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View
 ****
 endif::[]
 [id="{p}-{page_id}"]
-= Troubleshoot ECK
+= Troubleshooting ECK
 
 - <<{p}-common-problems>>
 - <<{p}-troubleshooting-methods>>
```

Comment:
<img width="651" alt="Capture d’écran 2024-10-25 à 15 49 19" src="https://github.com/user-attachments/assets/7a90f3ae-0672-4196-99b7-b03ec46cc949">

I wonder if we should add a link to this page here?
```suggestion
- <<{p}-troubleshooting-methods>>
- <<{p}-take-eck-dump>>
```

Also, could we rename the page in `run-eck-diagnostics`?